src/ExperiusPostcodeNLServiceProvider.php

```diff
@@ -11,7 +11,7 @@ public function boot()
         $this->loadViewsFrom(__DIR__.'/../resources/views', 'experius-postcode-nl');
 
         $this->publishes([
-            __DIR__.'/../resources/views' => resource_path('views/vendor/rapidez/experius-postcode-nl'),
-        ], 'views');
+            __DIR__.'/../resources/views' => resource_path('views/vendor/experius-postcode-nl'),
+        ], 'experius-postcode-nl-views');
```

Comment:
The group can just  be `views` so if anyone likes to publish all the views they can by publishing that group.